### DISCUSSION TITLE
176

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#141] [General]
# 潮流周刊第176期 - 车外风景

封面图拍摄来源于最近 51 收假回杭州路上随手拍，窗外属于浙江小城的风景，配上夕阳绿树还是很美的。

https://weekly.tw93.fun/posts/176-%E8%BD%A6%E5%A4%96%E9%A3%8E%E6%99%AF

**Comment by tsingwong** [comment]
最后这个就像之前多年前玩电脑游戏，那时候没钱只能买盗版碟，一张几块钱。现在也是买正版的。毕竟没有盈利的支持，游戏发行商是没有办法做到良性运转。游戏是这样的，软件也是这样的。

**Reply by tw93** [reply]
是的，好比有购买才会有更好的产品持续的出现

**Comment by Beeta** [comment]
提个问题，Pake打包 flomo有个 bug，点击左侧的标签列表无法展开。

**Reply by tw93** [reply]
谢谢反馈，下一个版本我来看看

**Reply by tw93** [reply]
方便提交一个issue吗？复现一下，我是了下我的 mac 机器是好的

**Reply by Beeta** [reply]
又测试了一下，是我的问题😂

**Comment by CY-Christin** [comment]
「让没有想法同时技术水平一般的程序员更加被动焦虑」，打人不打脸，骂人不揭短。

**Comment by haihong81** [comment]
想问下用了哪些免费软件替代？

**Reply by haihong81** [reply]
等回复中。。

**Reply by tw93** [reply]
推荐3个
todo软件直接用Mac的提醒事项来代替，稍微配置下很好用
录制的应用直接用QuickRecorder代替，够用也很轻量
大量的常用操作，快捷键工具全部用Raycast配置来做，完全够了

**Comment by Beeta** [comment]
「大量的常用操作，快捷键工具全部用Raycast配置来做，完全够了」这个可以展开讲讲吗～～

**Reply by tsingwong** [reply]
举几个例子，
- 之前可能需要用独立软件才能做到窗口管理；在 Raycast 中只需要 `left half` `bottom right quarter`  就能做窗口的管理
- mac 上有很多独立粘贴板软件；在 Raycast 中自带的粘贴板管理
- Raycast 有良好的插件系统，写一个文件上传 cdn 的小插件，代码量可能在 100 行左右就能完成
